docs/benchmarking.md

```diff
@@ -115,6 +115,67 @@ TODO(benvanik): Talk about VM Benchmarks
 IREE is instrumented with the C++ bindings from the
 [Google Web Tracing Framework](https://github.com/google/tracing-framework).
```

Comment:
Can remove references to WTF for now. The submodule and supporting code for WTF was removed when Tracy was added in https://github.com/google/iree/pull/1936.

Reply:
Removed, thanks!